packages/docs/docs/access/binary-security-context.md

```diff
@@ -96,7 +96,9 @@ const media = await medplum.createMedia({
   data: fileData,
   filename: 'patient-report.pdf',
   contentType: 'application/pdf',
-  subject: { reference: `Patient/${patientId}` },
+  additionalFields: {
+    subject: { reference: `Patient/${patientId}` },
+  }
```

Comment:
This example can/should be moved into the `examples` directory, so errors like this can be caught and constantly checked against the actual SDK code

Reply:
+1 . @finnbergquist let me know if you need help with the `MedplumCodeBlock`